### ISSUE TITLE
Hi, I am kinda confused about pseudo lidar points from depth map ?

### ISSUE THREAD

**Issue by nile649** [mileyan/pseudo_lidar#12]
can I use depth map from any architecture to create pseudo lidar points? GT lidar points are required to train model to predict pseudo lidar points from generated depth map?

**Comment by mileyan** [comment]
Yes, you can I use depth map from any architecture to create pseudo lidar points. But if you fine tune your model on the KITTI training set LIDAR points, you will get a better result. But the difference should be less than 5%.

**Comment by nile649** [comment]
My question is how can I create pseudo lidar points, if I don't have ground truth for the datasets. I am planning to test on Nyu2 datasets. Thanks

**Comment by mileyan** [comment]
You need the `calib`, `image_2`(left image) and `image_3`(right image) folders. And you need to generate the calibration files as the same format as KITTI object detection dataset.

**Comment by tareeqav** [comment]
@mileyan so when using a custom dataset, what transformation matrix should I use for `Tr_velo_to_cam` in the absence of any LiDAR at all?
